db/migrate/20230302202459_marketplace_encrypt_order_delivery_address.rb

```diff
@@ -0,0 +1,6 @@
+class MarketplaceEncryptOrderDeliveryAddress < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :marketplace_orders, :delivery_address, :deprecated_delivery_address
+    add_column :marketplace_orders, :delivery_address_ciphertext, :text
```

Comment:
why not encrypt the existing column like lockbox gem says is possible [here](https://github.com/ankane/lockbox#migrating-existing-data)?

Reply:
Ah, the migrating syntax is neat! I didn't realize it existed; but yea, that would save us the `rename_column` step and having to write the `find_each`!

I'll definitely do it next time!